x/bitcoin/types/tests/types_test.go

```diff
@@ -66,7 +66,7 @@ func TestOutPointInfo_Equals(t *testing.T) {
 }
 
 func TestNewMasterConsolidationAddress(t *testing.T) {
-	repeat := 100
+	repeat := 20
```

Comment:
this tests took ~15sec locally. If this change is not ok let me know @sammy1991106

Reply:
It's fine